### PR TITLE
Lesson 6, 7

### PR DESCRIPTION
**Pull request by KsuKuper**
Буду весьма благодарна и признательна за любой просмотр и комментарий!
Заходит трудно! Много времени уделяю! Потому прошу максимально подробно описывать мои недостатки и проблемы)
Благодарю!)

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -1,124 +1,291 @@
...
+    /*opacity: 1;*/
+}
+
+/* Page */
+.page {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 100%;
+    height: 100vh;
```
Comment (bogomolov-and) [inline styles/styles.css]: Use the `min-height` for this case. This will help you work with different screens in the future.

File: index.html [hunk 2]
```diff
@@ -2,329 +2,113 @@
...
-        </li>
-      </ul>
-    </section>
+        </div>
+      </div>
+    </div>
+  </div>
+    <!--    LESSON 5 -->
+    <!--    WHAT DO WE OFFER-->
+    <div class="cont">
```
Comment (bogomolov-and) [inline index.html]: Please, use Prettier for your code

File: styles/styles.css [hunk 3]
```diff
@@ -1,124 +1,291 @@
...
+    line-height: 1.6;
+    margin: 0;
 }
-h2 {
-   color: #1f4796;
+
+/*Lesson 5*/
+/*WHAT DO WE OFFER*/
+
+.cont {
```
Comment (bogomolov-and) [inline styles/styles.css]: Arrange the classes in the same order as in the html file

File: index.html [hunk 4]
```diff
@@ -2,329 +2,113 @@
...
+  <div class="page">
+    <div class="page__container">
+      <div class="page__container--inner">
+        <h1 class="page__container--title">Discover stays to live, work or just relax</h1>
+        <div class="page__container--subtitle">
+          <div class="hotel">Your destination or hotel name</div>
+          <div class="check">Check-in — Check-out</div>
+        </div>
+        <div class="page__container--search">
+          <div class="page__container--search-city">New York</div>
```
Comment (almuly) [inline index.html]: use input instead of divs

File: index.html [hunk 5]
```diff
@@ -2,329 +2,113 @@
...
-            <div class="advantage__desc">Support 24/7</div>
+    </div>
+  </header>
+  <div class="page">
+    <div class="page__container">
+      <div class="page__container--inner">
+        <h1 class="page__container--title">Discover stays to live, work or just relax</h1>
+        <div class="page__container--subtitle">
+          <div class="hotel">Your destination or hotel name</div>
+          <div class="check">Check-in — Check-out</div>
```
Comment (almuly) [inline index.html]: add label corresponding with input instead of div

File: index.html [hunk 6]
```diff
@@ -2,329 +2,113 @@
...
+        </div>
+        <div class="page__container--search">
+          <div class="page__container--search-city">New York</div>
+          <div class="page__container--search-inner-date">
+            <span class="page__container--search-date">Tue 15 Sept — Sat 19 Sept</span>
```
Comment (almuly) [inline index.html]: so, it will be value of input

File: index.html [hunk 7]
```diff
@@ -2,329 +2,113 @@
...
           </div>
-        </li>
-        <li class="advantage__item">
-          <div class="advantage__block">
-            <div class="advantage__icon">
-              <img src="./image/img_1.png" alt="icon-communicate" />
-            </div>
-            <div class="advantage__desc">Communicate directory</div>
+          <div class="page__container--search-set">2 Adults — 0 Children — 1 Room</div>
```
Comment (almuly) [inline index.html]: also, it will be value of input

File: styles/styles.css [hunk 8]
```diff
@@ -1,124 +1,291 @@
...
+    color: #383838;
 }
-input[type="text"]
-{
-    color: #e79898;
+
+/*FOOTER*/
+
+.flex-cont-footer {
+    background: #10367C;
```
Comment (almuly) [inline styles/styles.css]: wrap your footer in a block and transfer property background there

File: styles/styles.css [hunk 9]
```diff
@@ -1,124 +1,291 @@
...
+    max-width: 1440px;
+    height: 630px;
+    margin: 0;
```
Comment (almuly) [inline styles/styles.css]: lost auto. margin:0 auto

File: styles/styles.css [hunk 10]
```diff
@@ -1,124 +1,291 @@
...
+.cont_col_support {
+    width: 233px;
+    height: 122px;
+    margin: 120px 272px 0 0;
+    color: #ffffff;
 }
-.button {
-    min-width: 70px;
+
+.row_1.row_2.row_3 {
```
Comment (almuly) [inline styles/styles.css]: it's not necessary use names like row_1, row_2 , etc with the same property, just use row
Also, don't use number counting in names of classes, better to use semantic description, for exmple .footer_row, , .list_item and .etc

File: styles/styles.css [hunk 11]
```diff
@@ -1,124 +1,291 @@
...
-    margin-bottom: 50px;
+
+/* Header */
+.header {
+    width: 100%;
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    z-index: 1000;
```
Comment (bogomolov-and) [inline styles/styles.css]: Oh no, only without `position: absolute`
You can add a `div` wrapper and include the `header` and `form block` in it and add a background image for that `div` wrapper. Then you can delete `position: absolute` and everything connected with it

File: styles/styles.css [hunk 12]
```diff
@@ -1,124 +1,291 @@
...
+.cont_block_row {
+    display: flex;
 }
-ol::before {
-    content: " ! The first step towards the development !";
-    background-color: #d7d77b;
-    color: #ad3c3c;
-    font-weight: bold;
+
+.cont_col_about {
```
Comment (almuly) [inline styles/styles.css]: just use one class for columns in footer without fixed sizes. you can add width:33.3333% and align columns in footer using flex properties 
heigth and width will be achiving by the content and gaps in container ( margin, padding)

File: styles/styles.css [hunk 13]
```diff
@@ -1,124 +1,291 @@
...
+    margin: 120px 222px 0 104px;
+    color: #ffffff;
 }
-ol::after {
-    content: " ! The first step towards the development has been taken !";
-    background-color: #d7d77b;
-    color: #ad3c3c;
-    font-weight: bold;
+
+.cont_col_property {
```
Comment (almuly) [inline styles/styles.css]: the same

File: styles/styles.css [hunk 14]
```diff
@@ -1,124 +1,291 @@
...
+    width: 134px;
+    height: 310px;
+    margin: 120px 282px 0 0;
+    color: #ffffff;
 }
-.input {
-    min-inline-size: auto;
-    /*height: 100px;*/
+
+.cont_col_support {
```
Comment (almuly) [inline styles/styles.css]: also, use one universal class, if you need to add some spcial properties for items, you can use pseudoclasses

**Pull request closed without merging**